specs/resources/funding-sources/retrieveInitiateVerifyMicroDeposit.yml

```diff
@@ -60,3 +68,42 @@ post:
         application/vnd.dwolla.v1.hal+json:
           schema:
             type: object
+components:
+  requestBodies:
+    InitiateMicroDeposits:
+      title: InitiateMicroDeposits
+      description: Empty request body for initiating micro-deposits
+      type: object
+      nullable: true
+    VerifyMicroDeposits:
+      title: VerifyMicroDeposits
+      description: Request body for verifying micro-deposits
+      type: object
+      required:
+        - amount1
+        - amount2
+      properties:
+        amount1:
+          type: object
+          required:
+            - value
+            - currency
+          properties:
+            value:
+              type: string
+              example: 0.02
+            currency:
+              type: string
+              example: USD
+        amount2:
+          type: object
+          required:
+            - value
+            - currency
+          properties:
+            value:
+              type: string
+              example: 0.03
+            currency:
+              type: string
+              example: USD
```

Comment:
```suggestion
        amount1:
          type: object
          $ref: '../../schemas/commonSchemas.yml#/components/schemas/TransferAmount'
        amount2:
          type: object
          $ref: '../../schemas/commonSchemas.yml#/components/schemas/TransferAmount'
```

Reusing `TransferAmount` from commonSchemas. To be honest, `TransferAmount` should probably be renamed to something more generic—for example, `Amount`—similar to how its named in our SDKs.